### ISSUE TITLE
Release 0.19 throws an error when terraform 0.12 experimental variable_validation is enabled

### ISSUE THREAD

**Issue by mhulscher** [terraform-linters/tflint#881]
tflint 0.19.0 is said to support terraform 0.12 but throws an error when experimental `variable_validation` is enabled.

```hcl
terraform {
  experiments = [variable_validation]
}
```

```
$ tflint --version
TFLint version 0.19.0

$ tflint --deep
Failed to load configurations. 1 error(s) occurred:

Error: Experiment has concluded

  on main.tf line 3, in terraform:
   3:   experiments      = [variable_validation]

Experiment "variable_validation" is no longer available. Custom variable validation can now be used by default, without enabling an experiment.
```

**Comment by bendrucker** [comment]
Thanks for reporting, but unfortunately this is unavoidable, and not something we'll be able to change:

https://github.com/terraform-linters/tflint#limitations

> TFLint load configurations in the same way as Terraform v0.13. This means that it cannot inspect configurations that cannot be parsed on Terraform v0.13.

The description in the changelog is not terribly accurate. There is only backwards compatibility to the extent that the configuration would be supported in Terraform 0.13. Because the variable validation experiment has concluded, Terraform will return a parser error for any module that includes it. This means that modules that use it should also specify:


```tf
terraform {
  required_version = "~> 0.12"
}
```

The only workaround I can suggest would be to house this setting in an `experiments.tf` file (which I've always done) and then rename all `experiments.tf` files to `experiments.tf.bak` while running TFLint (and then undo after). It's cumbersome and won't work for modules you don't control. Another possibility is running `terraform 0.13upgrade` on modules before running TFLint and then undoing.
